Example/Firebase.xcodeproj/project.pbxproj

```diff
@@ -1914,6 +1919,9 @@
 		6003F581195388D10070C39A = {
 			isa = PBXGroup;
 			children = (
+				518855102231E09300CA4141 /* FIRMessagingExtensionHelperTest.m */,
+				5188550F2231E04C00CA4141 /* FIRMessagingClientTest.m */,
+				5188550D2231E02300CA4141 /* FIRMessagingExtensionHelperTest.m */,
```

Comment:
These 3 files are placed in the root directory of the Firebase Example project. Please move them into Messaging/.